mephisto/client/review/review_server.py

```diff
@@ -189,8 +189,7 @@ def next_task():
         )
 
         if output == "":
-            sys.stdout.write("{}\n".format(result))
-            sys.stdout.flush()
+            print("{}\n".format(result))
```

Comment:
You don't need the "\n" anymore since print auto-appends. To keep the functionality the same, let's remove that